### PR TITLE
feat: add announcements edit and delete functionality

### PR DESCRIPTION
**Pull request by ritikrajdev**
* Add functionality to edit and delete announcements

### CODE REVIEW

**Review by Abhishek-Kharpal — COMMENTED** [review]
LGTM

**Review by samim-hossain-mondal — APPROVED** [review]
LGTM

File: src/components/dsm/Announcements.js [hunk 1]
```diff
@@ -107,6 +117,46 @@ export default function Announcements() {
     }
   }
 
+  const handleEditAnnouncement = async (content) => {
+    try {
+      const reqBody = {
+        content,
+      }
+      const resData = await makeRequest(UPDATE_ANNOUNCMENT(projectId, editModalData.announcementId), { data: reqBody })
```
Comment (Kiranyadav1234) [inline src/components/dsm/Announcements.js:125]: editModalData?.announcementId

**Review by Kiranyadav1234 — APPROVED** [review]

File: src/components/dsm/Announcements.js (hunk 1 above)
Comment (ritikrajdev) [reply src/components/dsm/Announcements.js]: Hi Kiran, This has been resolved in [`20b707e` (#88)](https://github.com/tech-university-india/abd-frontend/pull/88/commits/20b707e292ff49aa4cb23e9eac5fb77b765b3b5a), really thanks for pointing it out 😄, will check on whole codebase for this fix once the thing is stable !!

**Pull request merged**